### ISSUE TITLE
Do not return anonymous actions for properties

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1878]
In case a property returns an `Action`, we should issue a warning if that action is an anonymous, as in following example:

```csharp
public Action SomeProperty
{
    get
    {
        return () => { };
    }
}
```

The reason is that each time the property returns a new created action object.

Same is valid for
```csharp
public Action SomeProperty => () => { };
```
and
```csharp
public Action SomeProperty
{
   get => () => { };
}
```
